wip-asdpc-minimum-mining-difficulty.md

```diff
@@ -0,0 +1,112 @@
+<pre>
+    WIP: WIP-asdpc-minimum-mining-difficulty
+    Layer: Consensus (hard fork)
+    Title: Set minimum mining difficulty 
+    Authors: Adán SDPC <adan@witnet.foundation>
+    Discussions-To: `#dev-general` channel on Witnet Community's Discord server
+    Status: Draft
+    Type: Standards Track
+    Created: 2021-04-08
+    License: BSD-2-Clause
+</pre>
+
+
+## Abstract
+
+This proposal introduces a lower bound on mining difficulty so that in case that the Active Reputation Set (ARS) goes to abnormally low sizes, mining is not taken over by the few identities therein, given the priority that their block candidates are given by [WIP-0009].
+
+## Motivation and rationale
+
+The Witnet protocol contains a series of rules that enable nodes to evaluate block candidates and deterministically and independently decide which one to consolidate (append to the chain) without any need for voting or coordination with other nodes.
+
+By design, these rules prioritized block candidates coming from identities whose reliability has been recently proven. This reliability is vetted by their presence in the Active Reputation Set (ARS). When comparing two block candidates, if one was created by an identity in the ARS, and the other does not, the one coming from the ARS member is always preferred.
```

Comment:
The only exception is that if the candidates from the ARS have not reputation and the outcomer yes